src/preprocess.py

```diff
@@ -366,64 +364,24 @@ def build_tasks(args):
             task.name, "test", preproc_dir)
         # When using pretrain_data_fraction, we need modified iterators for use
         # only on training datasets at pretraining time.
-        if args.pretrain_data_fraction < 1 and task.name in train_task_names:
```

Comment:
Just checking: This was simply broken before, right? (Since train_task_names doesn't exist.)

Reply:
Yes, it was broken. `train_task_names` should be `pretrain_task_names` unless I am missing something.